### PR TITLE
message-attributes support

### PR DESCRIPTION
**Pull request by rotemironsrc**

### CODE REVIEW

File: test/spec/unit-tests/Worker.ut.spec.js [hunk 1]
```diff
@@ -78,4 +79,51 @@ test('start - 1 invalid message with existing controller - should not call TEST_
...
+test('start - 1 valid message and atttributes array with existing controller - should call TEST_CONTROLLER1 handler', async (done) => {
+    const consumerStub    = new ConsumerMock();
+    const expectedMessage = {
+        type   : 'TEST_CONTROLLER1',
+        content: {
+            age: 19
+        }
+    };
+
+    var expectedMessageAttributes = {
```
Comment (ramimoshe) [inline test/spec/unit-tests/Worker.ut.spec.js]: const instead of var

**Comment by rotemironsrc** [comment]
add messageAttributes support

File: test/spec/unit-tests/Worker.ut.spec.js (hunk 1 above)
Comment (rotemironsrc) [reply test/spec/unit-tests/Worker.ut.spec.js]: done

File: src/consumers/SqsConsumer.js [hunk 2]
```diff
@@ -30,9 +30,12 @@ class SqsConsumer extends BaseConsumer {
 
     createConsumer(handler) {
         return Consumer.create({
+            messageAttributeNames  : ["All"],
             queueUrl     : this.queueUrl,
             sqs          : this.sqsClient,
-            handleMessage: async (message) => await handler(message.Body),
+            handleMessage: async (message) => {
```
[the remark this answers is not in the document]
Comment (rotemironsrc) [reply src/consumers/SqsConsumer.js]: done

**Pull request merged**